provision.sh

```diff
@@ -14,31 +14,47 @@ echo "---------------------------------------------"
 echo "Running vagrant provisioning"
 echo "---------------------------------------------"
 
+# install heroku toolbelt
+echo "-------------- Installing heroku toolbelt -------------------------"
+#wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
+# These shell script snippets are directly taken from heroku installation script
+# We want to avoid the apt-get update
+# add heroku repository to apt
+echo "deb http://toolbelt.heroku.com/ubuntu ./" > /etc/apt/sources.list.d/heroku.list
+# install heroku's release key for package verification
+wget -O- https://toolbelt.heroku.com/apt/release.key | apt-key add -
+
 echo "------------- Updating packages and their dependencies -----------"
 apt-get update -y # no need for sudo, and -y is needed to bypass the yes-no
+
+# install the toolbelt
+apt-get install heroku-toolbelt --no-install-recommends -y
```

Comment:
@alayek how many MB did you save with this. What do the recommended packages include? Will it negatively influence a typical project?